odoo/tools/xml_utils.py

```diff
@@ -104,7 +104,7 @@ def cleanup_xml_node(xml_node_or_string, remove_blank_text=True, remove_blank_no
     if isinstance(xml_node, str):
         xml_node = xml_node.encode()  # misnomer: fromstring actually reads bytes
     if isinstance(xml_node, bytes):
-        xml_node = etree.fromstring(xml_node)
+        xml_node = etree.fromstring(xml_node, etree.XMLParser(recover=True, resolve_entities=False))
```

Comment:
@odoo/rd-security Usage of a custom XMLParser. `resolve_entities` is explicitely set to `False`.